.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thanks for contributing! -->
+[Issue Key](https://jira.mongodb.org/browse/%7BISSUE_KEY%7D)
```

Comment:
I'm confused by this line. We don't include a hyperlink...

Reply:
I felt that it could be useful to include a link to the jira ticket in the PR body directly. Sometimes tickets have extra context in the comments and this would make it easier for reviews to get to the jira ticket (esp since putting the ticket in the title isn't a hyper link to jira itself)